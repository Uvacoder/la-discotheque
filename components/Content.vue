<template>
  <div class="Content">
    <Logo />
    <slot />
  </div>
</template>

<script>
import Logo from '~/components/Logo.vue'

export default {
  components: {
    Logo,
  }
}
</script>

<style>
.Content {
  margin-left: 96px;
  padding: 0 1em 0 26px;
}

.Content .Logo {
  margin: 2em 0 4em 0;
}
</style>
