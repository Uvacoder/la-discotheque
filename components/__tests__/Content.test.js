import { shallowMount } from '@vue/test-utils'

import Content from '~/components/Content.vue'

describe('Content.vue', () => {
  it('renders <slot> properly', () => {
    const wrapper = shallowMount(Content, {
      slots: {
        default: ['Hello world']
      }
    })
    expect(wrapper.find('slot'))
  })

  it('matches snapshot', () => {
    const wrapper = shallowMount(Content)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
