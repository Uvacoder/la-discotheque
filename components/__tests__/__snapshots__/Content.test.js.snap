// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content.vue matches snapshot 1`] = `<div role="main" class="Content"></div>`;
